markets/agave/reservesConfigs.ts

```diff
@@ -16,7 +16,7 @@ export const strategyBUSD: IReserveParams = {
   stableBorrowRateEnabled: false,
   reserveDecimals: '18',
   aTokenImpl: eContractid.AToken,
-  reserveFactor: '1000'
+  reserveFactor: '1000',
```

Comment:
Can you break out the linting changes into their own PR so it's easier to see what is happening in this one?